File: symbol/dimgui_style.ts
const ImVec2 = ["f32", "f32"] as const;
const symbols = {
  DImGuiStyleGetAlpha: {
    parameters: ["pointer"],
    result: "f32",
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleSetAlpha: {
    parameters: ["pointer", "f32"],
    result: "void",
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleGetDisabledAlpha: {
    parameters: ["pointer"],
    result: "f32",
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleSetDisabledAlpha: {
    parameters: ["pointer", "f32"],
    result: "void",
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleGetWindowPadding: {
    parameters: ["pointer"],
    result: { struct: ImVec2 },
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleSetWindowPadding: {
    parameters: ["pointer", { struct: ImVec2 }],
    result: "void",
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleGetWindowRounding: {
    parameters: ["pointer"],
    result: "f32",
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleSetWindowRounding: {
    parameters: ["pointer", "f32"],
    result: "void",
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleGetWindowBorderSize: {
    parameters: ["pointer"],
    result: "f32",
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleSetWindowBorderSize: {
    parameters: ["pointer", "f32"],
    result: "void",
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleGetWindowMinSize: {
    parameters: ["pointer"],
    result: { struct: ImVec2 },
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleSetWindowMinSize: {
    parameters: ["pointer", { struct: ImVec2 }],
    result: "void",
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleGetWindowTitleAlign: {
    parameters: ["pointer"],
    result: { struct: ImVec2 },
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleSetWindowTitleAlign: {
    parameters: ["pointer", { struct: ImVec2 }],
    result: "void",
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleGetWindowMenuButtonPosition: {
    parameters: ["pointer"],
    result: "i32",
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleSetWindowMenuButtonPosition: {
    parameters: ["pointer", "i32"],
    result: "void",
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleGetChildRounding: {
    parameters: ["pointer"],
    result: "f32",
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleSetChildRounding: {
    parameters: ["pointer", "f32"],
    result: "void",
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleGetChildBorderSize: {
    parameters: ["pointer"],
    result: "f32",
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleSetChildBorderSize: {
    parameters: ["pointer", "f32"],
    result: "void",
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleGetPopupRounding: {
    parameters: ["pointer"],
    result: "f32",
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleSetPopupRounding: {
    parameters: ["pointer", "f32"],
    result: "void",
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleGetPopupBorderSize: {
    parameters: ["pointer"],
    result: "f32",
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleSetPopupBorderSize: {
    parameters: ["pointer", "f32"],
    result: "void",
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleGetFramePadding: {
    parameters: ["pointer"],
    result: { struct: ImVec2 },
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleSetFramePadding: {
    parameters: ["pointer", { struct: ImVec2 }],
    result: "void",
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleGetFrameRounding: {
    parameters: ["pointer"],
    result: "f32",
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleSetFrameRounding: {
    parameters: ["pointer", "f32"],
    result: "void",
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleGetFrameBorderSize: {
    parameters: ["pointer"],
    result: "f32",
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleSetFrameBorderSize: {
    parameters: ["pointer", "f32"],
    result: "void",
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleGetItemSpacing: {
    parameters: ["pointer"],
    result: { struct: ImVec2 },
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleSetItemSpacing: {
    parameters: ["pointer", { struct: ImVec2 }],
    result: "void",
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleGetItemInnerSpacing: {
    parameters: ["pointer"],
    result: { struct: ImVec2 },
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleSetItemInnerSpacing: {
    parameters: ["pointer", { struct: ImVec2 }],
    result: "void",
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleGetCellPadding: {
    parameters: ["pointer"],
    result: { struct: ImVec2 },
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleSetCellPadding: {
    parameters: ["pointer", { struct: ImVec2 }],
    result: "void",
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleGetTouchExtraPadding: {
    parameters: ["pointer"],
    result: { struct: ImVec2 },
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleSetTouchExtraPadding: {
    parameters: ["pointer", { struct: ImVec2 }],
    result: "void",
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleGetIndentSpacing: {
    parameters: ["pointer"],
    result: "f32",
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleSetIndentSpacing: {
    parameters: ["pointer", "f32"],
    result: "void",
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleGetColumnsMinSpacing: {
    parameters: ["pointer"],
    result: "f32",
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleSetColumnsMinSpacing: {
    parameters: ["pointer", "f32"],
    result: "void",
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleGetScrollbarSize: {
    parameters: ["pointer"],
    result: "f32",
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleSetScrollbarSize: {
    parameters: ["pointer", "f32"],
    result: "void",
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleGetScrollbarRounding: {
    parameters: ["pointer"],
    result: "f32",
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleSetScrollbarRounding: {
    parameters: ["pointer", "f32"],
    result: "void",
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleGetGrabMinSize: {
    parameters: ["pointer"],
    result: "f32",
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleSetGrabMinSize: {
    parameters: ["pointer", "f32"],
    result: "void",
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleGetGrabRounding: {
    parameters: ["pointer"],
    result: "f32",
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleSetGrabRounding: {
    parameters: ["pointer", "f32"],
    result: "void",
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleGetLogSliderDeadzone: {
    parameters: ["pointer"],
    result: "f32",
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleSetLogSliderDeadzone: {
    parameters: ["pointer", "f32"],
    result: "void",
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleGetTabRounding: {
    parameters: ["pointer"],
    result: "f32",
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleSetTabRounding: {
    parameters: ["pointer", "f32"],
    result: "void",
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleGetTabBorderSize: {
    parameters: ["pointer"],
    result: "f32",
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleSetTabBorderSize: {
    parameters: ["pointer", "f32"],
    result: "void",
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleGetTabMinWidthForCloseButton: {
    parameters: ["pointer"],
    result: "f32",
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleSetTabMinWidthForCloseButton: {
    parameters: ["pointer", "f32"],
    result: "void",
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleGetColorButtonPosition: {
    parameters: ["pointer"],
    result: "i32",
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleSetColorButtonPosition: {
    parameters: ["pointer", "i32"],
    result: "void",
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleGetButtonTextAlign: {
    parameters: ["pointer"],
    result: { struct: ImVec2 },
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleSetButtonTextAlign: {
    parameters: ["pointer", { struct: ImVec2 }],
    result: "void",
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleGetSelectableTextAlign: {
    parameters: ["pointer"],
    result: { struct: ImVec2 },
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleSetSelectableTextAlign: {
    parameters: ["pointer", { struct: ImVec2 }],
    result: "void",
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleGetDisplayWindowPadding: {
    parameters: ["pointer"],
    result: { struct: ImVec2 },
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleSetDisplayWindowPadding: {
    parameters: ["pointer", { struct: ImVec2 }],
    result: "void",
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleGetDisplaySafeAreaPadding: {
    parameters: ["pointer"],
    result: { struct: ImVec2 },
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleSetDisplaySafeAreaPadding: {
    parameters: ["pointer", { struct: ImVec2 }],
    result: "void",
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleGetMouseCursorScale: {
    parameters: ["pointer"],
    result: "f32",
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleSetMouseCursorScale: {
    parameters: ["pointer", "f32"],
    result: "void",
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleGetAntiAliasedLines: {
    parameters: ["pointer"],
    result: "bool",
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleSetAntiAliasedLines: {
    parameters: ["pointer", "bool"],
    result: "void",
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleGetAntiAliasedLinesUseTex: {
    parameters: ["pointer"],
    result: "bool",
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleSetAntiAliasedLinesUseTex: {
    parameters: ["pointer", "bool"],
    result: "void",
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleGetAntiAliasedFill: {
    parameters: ["pointer"],
    result: "bool",
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleSetAntiAliasedFill: {
    parameters: ["pointer", "bool"],
    result: "void",
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleGetCurveTessellationTol: {
    parameters: ["pointer"],
    result: "f32",
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleSetCurveTessellationTol: {
    parameters: ["pointer", "f32"],
    result: "void",
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleGetCircleTessellationMaxError: {
    parameters: ["pointer"],
    result: "f32",
  } as const satisfies Deno.ForeignFunction,
  DImGuiStyleSetCircleTessellationMaxError: {
    parameters: ["pointer", "f32"],
    result: "void",
  } as const satisfies Deno.ForeignFunction,
} as const satisfies Deno.ForeignLibraryInterface;
export default symbols;

File: imgui/src/dimgui_style.cpp
#include "../dim_config.h"

// use sturcts without constructors
#define CIMGUI_DEFINE_ENUMS_AND_STRUCTS
#include "cimgui/cimgui.h"

// clang-format off
DIMGUI_EXPORT float DImGuiStyleGetAlpha (ImGuiStyle* s){ return s->Alpha;}
DIMGUI_EXPORT float DImGuiStyleGetDisabledAlpha (ImGuiStyle* s){ return s->DisabledAlpha;}
DIMGUI_EXPORT ImVec2 DImGuiStyleGetWindowPadding (ImGuiStyle* s){ return s->WindowPadding;}
DIMGUI_EXPORT float DImGuiStyleGetWindowRounding (ImGuiStyle* s){ return s->WindowRounding;}
DIMGUI_EXPORT float DImGuiStyleGetWindowBorderSize (ImGuiStyle* s){ return s->WindowBorderSize;}
DIMGUI_EXPORT ImVec2 DImGuiStyleGetWindowMinSize (ImGuiStyle* s){ return s->WindowMinSize;}
DIMGUI_EXPORT ImVec2 DImGuiStyleGetWindowTitleAlign (ImGuiStyle* s){ return s->WindowTitleAlign;}
DIMGUI_EXPORT ImGuiDir DImGuiStyleGetWindowMenuButtonPosition (ImGuiStyle* s){ return s->WindowMenuButtonPosition;}
DIMGUI_EXPORT float DImGuiStyleGetChildRounding (ImGuiStyle* s){ return s->ChildRounding;}
DIMGUI_EXPORT float DImGuiStyleGetChildBorderSize (ImGuiStyle* s){ return s->ChildBorderSize;}
DIMGUI_EXPORT float DImGuiStyleGetPopupRounding (ImGuiStyle* s){ return s->PopupRounding;}
DIMGUI_EXPORT float DImGuiStyleGetPopupBorderSize (ImGuiStyle* s){ return s->PopupBorderSize;}
DIMGUI_EXPORT ImVec2 DImGuiStyleGetFramePadding (ImGuiStyle* s){ return s->FramePadding;}
DIMGUI_EXPORT float DImGuiStyleGetFrameRounding (ImGuiStyle* s){ return s->FrameRounding;}
DIMGUI_EXPORT float DImGuiStyleGetFrameBorderSize (ImGuiStyle* s){ return s->FrameBorderSize;}
DIMGUI_EXPORT ImVec2 DImGuiStyleGetItemSpacing (ImGuiStyle* s){ return s->ItemSpacing;}
DIMGUI_EXPORT ImVec2 DImGuiStyleGetItemInnerSpacing (ImGuiStyle* s){ return s->ItemInnerSpacing;}
DIMGUI_EXPORT ImVec2 DImGuiStyleGetCellPadding (ImGuiStyle* s){ return s->CellPadding;}
DIMGUI_EXPORT ImVec2 DImGuiStyleGetTouchExtraPadding (ImGuiStyle* s){ return s->TouchExtraPadding;}
DIMGUI_EXPORT float DImGuiStyleGetIndentSpacing (ImGuiStyle* s){ return s->IndentSpacing;}
DIMGUI_EXPORT float DImGuiStyleGetColumnsMinSpacing (ImGuiStyle* s){ return s->ColumnsMinSpacing;}
DIMGUI_EXPORT float DImGuiStyleGetScrollbarSize (ImGuiStyle* s){ return s->ScrollbarSize;}
DIMGUI_EXPORT float DImGuiStyleGetScrollbarRounding (ImGuiStyle* s){ return s->ScrollbarRounding;}
DIMGUI_EXPORT float DImGuiStyleGetGrabMinSize (ImGuiStyle* s){ return s->GrabMinSize;}
DIMGUI_EXPORT float DImGuiStyleGetGrabRounding (ImGuiStyle* s){ return s->GrabRounding;}
DIMGUI_EXPORT float DImGuiStyleGetLogSliderDeadzone (ImGuiStyle* s){ return s->LogSliderDeadzone;}
DIMGUI_EXPORT float DImGuiStyleGetTabRounding (ImGuiStyle* s){ return s->TabRounding;}
DIMGUI_EXPORT float DImGuiStyleGetTabBorderSize (ImGuiStyle* s){ return s->TabBorderSize;}
DIMGUI_EXPORT float DImGuiStyleGetTabMinWidthForCloseButton (ImGuiStyle* s){ return s->TabMinWidthForCloseButton;}
DIMGUI_EXPORT ImGuiDir DImGuiStyleGetColorButtonPosition (ImGuiStyle* s){ return s->ColorButtonPosition;}
DIMGUI_EXPORT ImVec2 DImGuiStyleGetButtonTextAlign (ImGuiStyle* s){ return s->ButtonTextAlign;}
DIMGUI_EXPORT ImVec2 DImGuiStyleGetSelectableTextAlign (ImGuiStyle* s){ return s->SelectableTextAlign;}
DIMGUI_EXPORT ImVec2 DImGuiStyleGetDisplayWindowPadding (ImGuiStyle* s){ return s->DisplayWindowPadding;}
DIMGUI_EXPORT ImVec2 DImGuiStyleGetDisplaySafeAreaPadding (ImGuiStyle* s){ return s->DisplaySafeAreaPadding;}
DIMGUI_EXPORT float DImGuiStyleGetMouseCursorScale (ImGuiStyle* s){ return s->MouseCursorScale;}
DIMGUI_EXPORT bool DImGuiStyleGetAntiAliasedLines (ImGuiStyle* s){ return s->AntiAliasedLines;}
DIMGUI_EXPORT bool DImGuiStyleGetAntiAliasedLinesUseTex (ImGuiStyle* s){ return s->AntiAliasedLinesUseTex;}
DIMGUI_EXPORT bool DImGuiStyleGetAntiAliasedFill (ImGuiStyle* s){ return s->AntiAliasedFill;}
DIMGUI_EXPORT float DImGuiStyleGetCurveTessellationTol (ImGuiStyle* s){ return s->CurveTessellationTol;}
DIMGUI_EXPORT float DImGuiStyleGetCircleTessellationMaxError (ImGuiStyle* s){ return s->CircleTessellationMaxError;}




DIMGUI_EXPORT void DImGuiStyleSetAlpha (ImGuiStyle* s, float value){ s->Alpha=value;}
DIMGUI_EXPORT void DImGuiStyleSetDisabledAlpha (ImGuiStyle* s, float value){ s->DisabledAlpha=value;}
DIMGUI_EXPORT void DImGuiStyleSetWindowPadding (ImGuiStyle* s, ImVec2 value){ s->WindowPadding=value;}
DIMGUI_EXPORT void DImGuiStyleSetWindowRounding (ImGuiStyle* s, float value){ s->WindowRounding=value;}
DIMGUI_EXPORT void DImGuiStyleSetWindowBorderSize (ImGuiStyle* s, float value){ s->WindowBorderSize=value;}
DIMGUI_EXPORT void DImGuiStyleSetWindowMinSize (ImGuiStyle* s, ImVec2 value){ s->WindowMinSize=value;}
DIMGUI_EXPORT void DImGuiStyleSetWindowTitleAlign (ImGuiStyle* s, ImVec2 value){ s->WindowTitleAlign=value;}
DIMGUI_EXPORT void DImGuiStyleSetWindowMenuButtonPosition (ImGuiStyle* s, ImGuiDir value){ s->WindowMenuButtonPosition=value;}
DIMGUI_EXPORT void DImGuiStyleSetChildRounding (ImGuiStyle* s, float value){ s->ChildRounding=value;}
DIMGUI_EXPORT void DImGuiStyleSetChildBorderSize (ImGuiStyle* s, float value){ s->ChildBorderSize=value;}
DIMGUI_EXPORT void DImGuiStyleSetPopupRounding (ImGuiStyle* s, float value){ s->PopupRounding=value;}
DIMGUI_EXPORT void DImGuiStyleSetPopupBorderSize (ImGuiStyle* s, float value){ s->PopupBorderSize=value;}
DIMGUI_EXPORT void DImGuiStyleSetFramePadding (ImGuiStyle* s, ImVec2 value){ s->FramePadding=value;}
DIMGUI_EXPORT void DImGuiStyleSetFrameRounding (ImGuiStyle* s, float value){ s->FrameRounding=value;}
DIMGUI_EXPORT void DImGuiStyleSetFrameBorderSize (ImGuiStyle* s, float value){ s->FrameBorderSize=value;}
DIMGUI_EXPORT void DImGuiStyleSetItemSpacing (ImGuiStyle* s, ImVec2 value){ s->ItemSpacing=value;}
DIMGUI_EXPORT void DImGuiStyleSetItemInnerSpacing (ImGuiStyle* s, ImVec2 value){ s->ItemInnerSpacing=value;}
DIMGUI_EXPORT void DImGuiStyleSetCellPadding (ImGuiStyle* s, ImVec2 value){ s->CellPadding=value;}
DIMGUI_EXPORT void DImGuiStyleSetTouchExtraPadding (ImGuiStyle* s, ImVec2 value){ s->TouchExtraPadding=value;}
DIMGUI_EXPORT void DImGuiStyleSetIndentSpacing (ImGuiStyle* s, float value){ s->IndentSpacing=value;}
DIMGUI_EXPORT void DImGuiStyleSetColumnsMinSpacing (ImGuiStyle* s, float value){ s->ColumnsMinSpacing=value;}
DIMGUI_EXPORT void DImGuiStyleSetScrollbarSize (ImGuiStyle* s, float value){ s->ScrollbarSize=value;}
DIMGUI_EXPORT void DImGuiStyleSetScrollbarRounding (ImGuiStyle* s, float value){ s->ScrollbarRounding=value;}
DIMGUI_EXPORT void DImGuiStyleSetGrabMinSize (ImGuiStyle* s, float value){ s->GrabMinSize=value;}
DIMGUI_EXPORT void DImGuiStyleSetGrabRounding (ImGuiStyle* s, float value){ s->GrabRounding=value;}
DIMGUI_EXPORT void DImGuiStyleSetLogSliderDeadzone (ImGuiStyle* s, float value){ s->LogSliderDeadzone=value;}
DIMGUI_EXPORT void DImGuiStyleSetTabRounding (ImGuiStyle* s, float value){ s->TabRounding=value;}
DIMGUI_EXPORT void DImGuiStyleSetTabBorderSize (ImGuiStyle* s, float value){ s->TabBorderSize=value;}
DIMGUI_EXPORT void DImGuiStyleSetTabMinWidthForCloseButton (ImGuiStyle* s, float value){ s->TabMinWidthForCloseButton=value;}
DIMGUI_EXPORT void DImGuiStyleSetColorButtonPosition (ImGuiStyle* s, ImGuiDir value){ s->ColorButtonPosition=value;}
DIMGUI_EXPORT void DImGuiStyleSetButtonTextAlign (ImGuiStyle* s, ImVec2 value){ s->ButtonTextAlign=value;}
DIMGUI_EXPORT void DImGuiStyleSetSelectableTextAlign (ImGuiStyle* s, ImVec2 value){ s->SelectableTextAlign=value;}
DIMGUI_EXPORT void DImGuiStyleSetDisplayWindowPadding (ImGuiStyle* s, ImVec2 value){ s->DisplayWindowPadding=value;}
DIMGUI_EXPORT void DImGuiStyleSetDisplaySafeAreaPadding (ImGuiStyle* s, ImVec2 value){ s->DisplaySafeAreaPadding=value;}
DIMGUI_EXPORT void DImGuiStyleSetMouseCursorScale (ImGuiStyle* s, float value){ s->MouseCursorScale=value;}
DIMGUI_EXPORT void DImGuiStyleSetAntiAliasedLines (ImGuiStyle* s, bool value){ s->AntiAliasedLines=value;}
DIMGUI_EXPORT void DImGuiStyleSetAntiAliasedLinesUseTex (ImGuiStyle* s, bool value){ s->AntiAliasedLinesUseTex=value;}
DIMGUI_EXPORT void DImGuiStyleSetAntiAliasedFill (ImGuiStyle* s, bool value){ s->AntiAliasedFill=value;}
DIMGUI_EXPORT void DImGuiStyleSetCurveTessellationTol (ImGuiStyle* s, float value){ s->CurveTessellationTol=value;}
DIMGUI_EXPORT void DImGuiStyleSetCircleTessellationMaxError (ImGuiStyle* s, float value){ s->CircleTessellationMaxError=value;}

// clang-format on